testserver/__manifest__.py

```diff
@@ -4,19 +4,13 @@
     'name': 'Testserver',
     'version': '0.1',
     'summary': 'Testserver',
-    'description': """
-Mark this server as a testserver!
-=================================
-So that it is not confused with the real one
-    """,
     'author': 'Sunflower IT',
     'website': 'http://sunflowerweb.nl',
     'category': 'Specific Industry Applications',
     'sequence': 0,
     'depends': ['base', 'web'],
```

Comment:
Dont have to depend on base, [web already does](https://github.com/OCA/OCB/blob/10.0/addons/web/__manifest__.py#L15)

Reply:
fixed